chart/otomi/templates/job.yaml

```diff
@@ -36,6 +36,7 @@ spec:
             - |
               kubectl create ns otomi &> /dev/null
               binzx/otomi bootstrap && binzx/otomi apply
+              binzx/otomi apply-e2e
```

Comment:
Running this command after the apply,  prevents from seeing the final message with the Otomi url and access